rpm_spec/subpackages/manageiq-core

```diff
@@ -25,7 +25,7 @@ Requires: socat
 %posttrans core
 # 'bin' needs to be copied, not symlinked
 [[ -e /var/www/miq/vmdb/bin ]] && rm -rf /var/www/miq/vmdb/bin
-cp -r %{gemset_root}/vmdb/bin /var/www/miq/vmdb/bin
+cp -a %{gemset_root}/vmdb/bin /var/www/miq/vmdb/bin
```

Comment:
This was the only command that was `cp` not `%{__cp}` but didn't want to change that without understanding if that was intentional

Reply:
I think because it's in `posttrans` it is actually running as a proper shell script and not an rpm macro. I don't understand why it's just not double-copied during install time though.  Perhaps that's something we can change afterwards

Reply:
we tend to copy instead of putting them down in the rpm itself.
this is remnant from our git on appliance days.

Would be very cool if we could just use rpm to install in the right spot in the first place